applications/ConstitutiveLawsApplication/custom_constitutive/hyper_elastic_isotropic_quasi_incompressible_isochoric_neo_hookean_3d.cpp

```diff
@@ -0,0 +1,445 @@
+    v[59]=-(v[47]*v[58]);
+    v[178]=v[38]*v[59];
+    v[176]=v[135]*v[59];
+    v[163]=v[46]*v[59];
+    v[48]=-(v[46]*v[47]);
+    v[146]=v[38]*v[48];
+    v[40]=1e0*v[201]*v[38];
+    v[39]=-(v[38]*v[47]);
+    v[174]=v[154]*v[39];
+    v[99]=v[1]/v[14]+v[163];
+    v[63]=v[178]+v[5]/v[14];
+    v[54]=v[146]+v[9]/v[14];
+    v[10]=C1;
+    v[11]=Pressure;
+    v[226]=v[11]*v[40];
+    v[218]=v[11]*v[60];
+    v[217]=-(v[11]*v[84]);
+    v[216]=v[11]*v[76];
+    v[214]=v[11]*v[68];
+    v[212]=v[11]*v[50];
+    v[12]=std::pow(v[14],0.5e0);
+    v[202]=-(v[11]*v[12]);
+    v[203]=((-4e0/3e0)*v[10])/std::pow(v[12],5.0*one_third);
+    v[86]=v[203]*v[84];
+    v[78]=v[203]*v[76];
+    v[70]=v[203]*v[68];
+    v[62]=v[203]*v[60];
+    v[53]=v[203]*v[50];
+    v[42]=v[203]*v[40];
+    v[31]=(2.0*v[10])/std::pow(v[12],2.0*one_third);
+    v[138]=-one_third*v[31];
+    v[13]=v[1]+v[5]+v[9];
+    v[207]=1e0*v[13];
+    v[204]=-one_third*v[13];
+    v[205]=1e0*v[204];
+    v[225]=v[217]+v[204]*v[86];
+    v[148]=v[204]*v[78];
+    v[229]=v[148]-v[216];
+    v[145]=v[205]*v[70];
+    v[224]=v[145]-v[214];
+    v[142]=v[138]+v[205]*v[62];
+    v[227]=v[142]+v[218];
+    v[139]=v[138]+1e0*v[205]*v[53];
+    v[223]=v[139]+v[212];
+    v[134]=v[138]+1e0*v[205]*v[42];
+    v[220]=v[134]+v[226];
+    v[117]=1e0*v[205]*v[99];
+    v[114]=1e0*v[205]*v[63];
+    v[93]=1e0*v[205]*v[54];
+    v[34]=1e0*v[205]*v[31];
+    v[222]=v[34]*v[48];
+    v[219]=v[34]*v[58];
+    v[215]=v[34]*v[46];
+    v[211]=v[34]*v[38];
+    v[228]=v[34]*v[59];
+    v[221]=v[34]*v[39];
+    v[208]=v[202]+v[34];
+    v[15]=v[38]/v[14];
+    v[206]=-(v[11]*v[15]);
+    v[210]=v[206]*v[60]+v[202]*v[63];
+    v[209]=v[206]*v[50]+v[202]*v[54];
+    v[55]=-one_third*v[15];
+    v[44]=1e0+v[207]*v[55];
+    v[17]=v[135]/v[14];
+    v[18]=v[167]/v[14];
+    v[20]=v[46]/v[14];
+    v[213]=-(v[20]*v[218])+v[202]*v[99];
+    v[96]=-one_third*v[20];
+    v[91]=1e0+v[207]*v[96];
+    v[21]=v[154]/v[14];
+    v[24]=v[58]/v[14];
+    v[116]=-one_third*v[24];
+    v[112]=1e0+1e0*v[116]*v[207];
+    if (rFlags.Is(ConstitutiveLaw::COMPUTE_STRESS)) {
+        rStress[0]=v[12]*v[206]+v[31]*v[44];
+        rStress[1]=v[20]*v[202]+v[31]*v[91];
+        rStress[2]=v[202]*v[24]+v[112]*v[31];
+        rStress[3]=v[17]*v[208];
+        rStress[4]=v[208]*v[21];
+        rStress[5]=v[18]*v[208];
+    }
+    if (rFlags.Is(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR)) {
+        rTangentTensor(0,0)=2.0*(-(v[206]*v[40])+v[42]*v[44]+v[31]*(v[205]*v[38]*v[39]+v[55]));
+        rTangentTensor(0,1)=2.0*(v[209]+v[44]*v[53]+v[31]*(v[55]+v[93]));
+        rTangentTensor(0,2)=2.0*(v[210]+v[31]*(v[114]+v[55])+v[44]*v[62]);
+        rTangentTensor(0,3)=v[211]*v[67]-v[206]*v[68]+v[44]*v[70];
+        rTangentTensor(0,4)=v[206]*v[76]+v[44]*v[78]+v[208]*(v[152]-v[8]/v[14]);
+        rTangentTensor(0,5)=v[211]*v[83]-v[206]*v[84]+v[44]*v[86];
+        rTangentTensor(1,0)=2.0*(v[209]+v[42]*v[91]+v[31]*(v[93]+v[96]));
```

Comment:
```suggestion
        rTangentTensor(1,0)=2*(v[209]+v[42]*v[91]+v[31]*(v[93]+v[96]));
```
Sorry for repeating myself, a priori if we multiply by a integer instead of a double, then itis more probable that the compiler will optimize it